<launch>
    <node pkg="nhk_2020" type="pub_cmd" name="pub_cmd" output="screen">
    </node>
    <arg name="kp" default="1.5"/>
    <rosparam param="kp" subst_value="True">$(arg kp)</rosparam>

    <arg name="cnt" default="1"/>
    <rosparam param="cnt" subst_value="True">$(arg cnt)</rosparam>

    <arg name="ki" default="0"/>
    <rosparam param="ki" subst_value="True">$(arg ki)</rosparam>
</launch>
